staging/src/k8s.io/apiserver/pkg/server/options/priority_and_fairness.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"fmt"
+
+	"github.com/spf13/pflag"
+	utilflowcontrol "k8s.io/apiserver/pkg/util/flowcontrol"
+	"k8s.io/utils/path"
+)
+
+// PriorityAndFairnessOptions holds the api server priority and fairness options.
+type PriorityAndFairnessOptions struct {
+	// ConfigFile is the file path with priority and fairness configuration
+	ConfigFile string
+}
+
+// NewPriorityAndFairnessOptions creates a new instance of PriorityAndFairnessOptions
+//
+// The option is to point to a configuration file for priority and fairness.
+// It can override default values used there.
```

Comment:
I find this language a little confusing.  To what does "there" refer?

Reply:
Right, removed this sentence. By "there" I meant https://github.com/kubernetes/kubernetes/blob/1c594e7e01a899807431c806cd11c1d27c885c9c/staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/config.go#L66